tsinfer/formats.py

```diff
@@ -699,6 +717,13 @@ class SampleData(DataContainer):
     :param int chunk_size: The chunk size used for
         `zarr arrays <http://zarr.readthedocs.io/>`_. This affects
         compression level and algorithm performance. Default=1024.
+    :param int max_file_size: If a file is being used to store this data, set
+        a maximum size in bytes for the stored file. The default is to allow
+        relatively large files, so if you are running on a system with disk
```

Comment:
This is vague --- what's a relatively large file? We need to define what our default is, not force the user to figure it out.

Reply:
OK. The default is 2**30 on windows, or the zarr default otherwise. I'll modify the text accordingly.

Reply:
How about 

```
        If a file is being used to store this data, set
        a maximum size in bytes for the stored file. If None, the default value
        for `zarr.LMDBStore  <http://zarr.readthedocs.io/>` is used
        (except on windows, see note above). If you are running on a system with
        storage space constraints, you may wish to explictly set a value that will
        not use up your storage quota. Default=None (or 2**30 on windows systems).
```

Reply:
The GiB equivalent to 2**30 is in the note above. I decided to put it there as it's confusing to use different units in the `Default=xxx` section, although I could, I suppose, repeat this, i.e.  `Default=None (or 2**30, i.e. 1GiB, on windows systems).`

Re the zarr default, personally I'm more used to the upstream libraries like zarr deciding a sensible default for me, and it just seemed like less maintenance for us to simply rely on zarr setting something reasonable.  But I'm not that bothered either way. It [seems like](https://zarr.readthedocs.io/en/stable/_modules/zarr/storage.html) zarr defaults to `map_size = (2**40 if sys.maxsize >= 2**32 else 2**28)` which I guess we might want to copy?